src/gui/SetupDialog.cpp

```diff
@@ -1471,6 +1509,40 @@ void SetupDialog::setBackgroundArtwork( const QString & _ba )
 
 
 
+void SetupDialog::setAutoSaveInterval( int value )
+{
+	QString minutes = value > 1 ? tr( "minutes" ) : tr( "minute" );
+
+	if( m_saveIntervalSlider->value() != value )
+	{
+		m_saveIntervalSlider->setValue( value );
+	}
+
+	m_saveIntervalLbl->setText( tr( "Auto save interval: %1 %2" ).arg(
```

Comment:
Lesser of two evils.  I don't know enough languages to know if the word `minutes` always comes after the number, but we can let the translators attack that decision i.e. `10 Dollars` versus `$10`, etc.